Dockerfile

```
@@ -0,0 +1,24 @@
+FROM alpine:3.14.1
+
+EXPOSE 80
+WORKDIR /var/www/app
+
+RUN apk --no-cache --update add \
+    tzdata ca-certificates curl nginx s6  php7 php7-fpm php7-opcache php7-session php7-mbstring php7-pgsql \
```

```suggestion
    tzdata ca-certificates curl nginx s6 libpq php7 php7-fpm php7-opcache php7-session php7-mbstring php7-pgsql \
```

See dockage/phppgadmin#6.